### PR TITLE
introduce simpletest environment vars

### LINKED ISSUE #65: Add simpletest environment variables to lando config, and other local dev config files.
## Motivation
We would like to encourage people to author and run tests locally before making a PR, plus would like to document things with less friction, setup time.

### Is your feature request related to a problem? Please describe.
Currently every time I want to run a phpunit test, I have to set 
export SIMPLETEST_BASE_URL="http://localhost"
export SIMPLETEST_DB="mysql://drupal8:drupal8@database/drupal8"

### Describe the solution you'd like
Add simpletest environment variables to lando config, and other local dev config files.

### PR DESCRIPTION
**Pull request by trackleft**
Closes #65 

## Description
See #65 
## How Has This Been Tested?
locally
`lando rebuild -y && lando start && lando install`
`lando ssh`
`curl https://lando.lndo.site/`
`phpunit -c web/core/phpunit.xml.dist path/to/code`
`env` find variables set.

ddev worked much the same way.

### CODE REVIEW

File: .lando.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ services:
     composer:
       hirak/prestissimo: ^0.3.9
     overrides:
+      environment:
+        SIMPLETEST_DB: 'mysql://drupal8:drupal8@database/drupal8'
+        SIMPLETEST_BASE_URL: 'https://lando.lndo.site/'
```
Comment (tadean) [inline .lando.yml:16]: Should this be `http://localhost` similar to the ddev URL? There might be cases where `lando.lndo.site` might not be the valid URL of the site.

Comment (trackleft) [reply .lando.yml:16]: To be safe, I'll change it

File: .ddev/docker-compose.override.yaml [hunk 2]
```diff
@@ -2,17 +2,17 @@ version: '3.6'
...
 services:
   web:
+    environment:
+      SIMPLETEST_DB: 'mysql://db:db@db/db'
+      SIMPLETEST_BASE_URL: 'http://localhost'
     volumes:
       # Don't share our host working directory as /var/www/html. We want /var/www/html empty for composer.
       -  /var/www/html
-      # Instead share our host working directory as a standalone package.
+        # Instead share our host working directory as a standalone package.
```
Comment (joeparsons) [inline .ddev/docker-compose.override.yaml:11]: I think this comment was indented correctly before...

**Comment by joeparsons** [comment]
@joshuasosa Could you review the ddev changes in this PR when you have a chance?  (Sorry I commented on the wrong PR originally)

**Comment by tadean** [comment]
Worked successfully for me in the latest release of Lando:

```
www-data@21b54532f902:/app$ env | grep "SIMPLETEST"
SIMPLETEST_DB=mysql://drupal8:drupal8@database/drupal8
SIMPLETEST_BASE_URL=http://localhost
```

**Pull request merged**